.github/workflows/pr-agent.yml

```diff
@@ -1,20 +0,0 @@
-on:
```

Comment:
These code changes remove the workflow that was triggered on pull requests and issue comments. The workflow was running a job named "pr_agent_job" that checked if the event sender was not a bot and then ran the "pr-agent" action with certain permissions and environment variables. The code changes effectively remove this workflow from the configuration file.